heron/common/src/cpp/network/network_error.h

```diff
@@ -30,4 +33,12 @@ enum NetworkErrorCode {
   TIMEOUT
 };
 
+static inline std::ostream& operator<<(std::ostream& os, const NetworkErrorCode code) {
+  std::string strs[] = { "OK", "Connection error", "Read error", "Write error",
```

Comment:
this looks useful https://aantron.github.io/better-enums/tutorial/Conversions.html#Strings and easy to include in, if we bother to use it at all...